tests/pr_test.go

```diff
@@ -100,3 +100,29 @@ func TestRunCompleteUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunSecureSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  "solutions/secure",
+		Region:        "us-south",
+		Prefix:        "els-sr-da",
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"elasticsearch_version":      "8.10", // Always lock this test into the latest supported elasticsearch version
```

Comment:
hmm, I think don't pass any value here and allow it to always use the default ICD version